### ISSUE TITLE
Comments from David Kirkby (SAC) on 9.4 (Photometric Redshifts)

### ISSUE THREAD

**Issue by dkirkby** [LSSTScienceCollaborations/ObservingStrategy#633]
Looks good overall.  A few comments / suggestions below:

9.4.1:
 - Specifications could be more precise.  For example, is the average bias (3) calculated including the catastrophic failures (2)?  Are the requirements integrated over some n(z) or in predefined z bins?
- Add reference for Mahalanobis distance?

9.4.2:
- I guess the "interquartile range" identifies the catastrophic outliers, but this isn't very clear.

9.4.3:
- What are the advantages of the Euclid  simulation catalog over the LSST CatSim?
- Is i < 25.3 cut on true i or after adding simulated magnitude errors?
- Cannot really conclude that "u band is necessary to limit scatter" with this study since you have not demonstrated that favoring other bands wouldn't have a similar effect.
- Comment on the fact that sigma requirement is not being met after 10 years in Fig 9.14.

**Comment by MelissaGraham** [comment]
Hi David,
Thanks so much for this feedback. I made some changes based on your comments, and also some broader updates to reflect the progress that has been made on this project outside of this white paper. Responses to your comments are below, and I'll shortly initiate a pull request to integrate my changes. 
Melissa

> 9.4.1:
> Specifications could be more precise. For example, is the average bias (3)
> calculated including the catastrophic failures (2)? Are the requirements
> integrated over some n(z) or in predefined z bins?
In the "Specifications" section we have now written these as slightly more general, but this pretty much exactly how they are written in the SRD. Then, in the "Metrics" section, we go into greater detail about how we interpret and calculate each of them, so hopefully overall these two changes help to clarify the specifications and metrics. 

> Add reference for Mahalanobis distance?
Added reference to Mahalanobis (1936)

> 9.4.2:
> I guess the "interquartile range" identifies the catastrophic outliers, but
> this isn't very clear.
Indeed, we use the IQR to exclude catastrophic outliers from having too much influence. This is hopefully now better explained in the Metrics section.

> 9.4.3:
> What are the advantages of the Euclid simulation catalog over the LSST
> CatSim?
I've not actually compared them, so perhaps nothing... but note that I've updated the description of the catalog to be more specific about where it came from.

> Is i < 25.3 cut on true i or after adding simulated magnitude errors?
This has changed to an i<25.05 cut on true i, followed by a i<25 cut on the simulated observed magnitudes after scattering basedon their errors -- and this has been made clearer in the text as well.

> Cannot really conclude that "u band is necessary to limit scatter" with this
> study since you have not demonstrated that favoring other bands wouldn't
> have a similar effect.
Good point. I've changed the wording to say "We can also see that a lack of $u$-band data induces more scatter in the photo-$z$ results at $z<0.5$ and $z>2.0$, but that allotting more than 56 visits to $u$-band does not necessarily improve the results, since this comes at the expense of visits in the other filters."

> Comment on the fact that sigma requirement is not being met after 10 years
> in Fig 9.14.
In our case, this can be pinpointed to the choice of a percentage point function of 95%. Lowering to 68% creates a demonstration in which the SRD specifications are met at 10 years.

But more generally, we are not proposing this as the ``best" possible algorithm, and have pointed this out in the section under "Currently implemented photo-$z$ algorithm" by adding this sentence:
"Our adopted method is not necessarily the ``best" way to calculate photometric redshifts, but due to its relatively direct relation between the photometric errors and the uncertainty in the estimated photo-$z$, it is especially appropriate analyzing the impact of changes to the LSST observing strategy."
